rosdep/python.yaml

```diff
@@ -3412,6 +3412,10 @@ python-pynmea2:
   ubuntu:
     pip:
       packages: [pynmea2]
+python-pypcd-pip:
```

Comment:
A key named `python-pypcd-pip` implies a Python 2 version of the key.  Are you really looking for the Python 2 version, or the Python 3 one?

Reply:
Python 2 version